Task20/TestGoogle/UnitTest1.cs

```diff
@@ -0,0 +1,38 @@
+namespace TestGoogle;
+
+public class Tests
+{
+    IWebDriver _driver;
+
+    [SetUp]
+    public void Setup()
+    {
+        _driver = new ChromeDriver();
+        _driver.Url = "https://www.google.com";
+        _driver.Manage().Window.Maximize();
+    }
+
+    [Test]
+    public void Test1()
+    {
+        FrontPage frontPage = new FrontPage(_driver);
+        frontPage.PressTermsAndConditionsButton();
+        frontPage.PressLoginButton();
+        
+        LoginPage loginPage = new LoginPage(_driver);
+        loginPage.SetEmailAdress();
+        loginPage.PressEmailNextButton();
+        loginPage.SetPassword();
+        loginPage.PressPasswordNextButton();
+
+        FrontPageAfterLogIn frontPageAfterLogIn = new FrontPageAfterLogIn(_driver);
+
+        Assert.IsTrue(frontPageAfterLogIn.CheckAccountImage());
```

Comment:
Please, add the second parameter here to get custom message in case this test will fail at this line. For example, "Account Image is not displayed"